ui/src/app/components/header/header.tsx

```diff
@@ -25,52 +26,54 @@ export const Header = (props: {pageHasShortcuts: boolean; changeNamespace: (val:
         getVersion();
     }, []);
     React.useEffect(() => {
-      if (namespace && namespace != namespaceInfo.namespace) {
-        props.changeNamespace(namespace);
-        setNsInput(namespace);
-      }
+        if (namespace && namespace != namespaceInfo.namespace) {
+            props.changeNamespace(namespace);
+            setNsInput(namespace);
+        }
     }, []);
     return (
-        <GenericHeader>
-            <Link to='/'>
-                <Brand path={name} brandName='Argo Rollouts' logo={<Logo />} />
+        <header className='rollouts-header'>
+            <Link to='/' className='rollouts-header__brand'>
+                <Logo />
+                <div>
+                    <div className='rollouts-header__title'>
+                        <img src='/assets/images/argologo.svg' alt='Argo Text Logo' style={{filter: 'invert(100%)', height: '1em'}} />
```

Comment:
Can we make this source relative by dropping the beginning `/`? this will allow it to be found properly when it is embedded in the executable as well as ran in webpack